gui/default/untrusted/syncthing/device/editDeviceModalView.html

```diff
@@ -146,6 +146,10 @@
         </div>
       </div>
     </form>
+    <div class="form-group" ng-if="willBeReintroducedBy">
+      <label translate for="removal">Device Removal</label>
+      <p class="text-danger">When you remove this device, it might be automatically reintroduced by {{ willBeReintroducedBy }}.</p>
```

Comment:
Missing translate handling here

Reply:
Not entirely sure this merits `class="text-danger"` on the entire text. We don't have danger-colored texts in other places, do we? All in all the removal is harmless so I think the danger is minimal.

Reply:
@imsodin handled the translation, I just forgot to sync the untrusted files.